### PR TITLE
Add 19892

### PR DESCRIPTION
**Pull request by Getinet309**

### CODE REVIEW

**Comment by Getinet309** [comment]
Most of the texts contain the fourth ''ኣ'', but this should change to the first ''አ''
The text "አበ መኔት" should be changed to "አበ ምኔት"፣ "ወእምሰዐታታ" should be changed to "ወእምሰዐታተ"
፣"ወአርእተ" should be changed to "ወአርእስተ"
፣ "ኃበ ዐሞደ ወርቅ" should be changed to "ኀበ ዐምደ ወርቅ"፣ "ላዕለይ" to "ላዕላይ"፣ "አሞ" to "አመ"፣"ነሓሲ" to "ነሐሴ"፣"ነሐሲ" to "ነሐሴ"፣"ጰጉመን" to "ጳጉሜን"፣"ክብራን" to "ክቡራን"፣"ዘጥረዩ" to "ዘአጥረዩ"፣ "ወድርሰናት" to "ወድርሳናት"፣ "እሶት" to "እሳት"፣"ወእርምዖስኒ" to "ወኤርምያስኒ"። 
There is an unclear word at line 10143 which says "እምእምርሕት" it may be changed to "እምትምህርት"

**Comment by PietroLiuzzo** [comment]
@thea-m kindly asked to review this PR.

File: DabraWarq/DW03.xml [hunk 1]
```diff
@@ -8914,8 +8914,8 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 <incipit xml:lang="gez">
                                     <locus target="#197ra"/>
                                     <hi rend="rubric">አመ፡ ፳ወ፭ለተሙዝ፡ <gap reason="lost"/> በዕብብራውያን፡
-                                        በስመ፡ እግዚአብሔር፡ ልዑል፡ ብሑ</hi>ተ፡ ህሉና፡ ወስልጣን። ዝንቱ፡ ወርኃ፡ ነሓሲ፡
-                                    ወውእቱ፡ ፲ወ፪እምኆልቈ፡ አውራኅ<hi rend="rubric">፨</hi>
+                                        በስመ፡ እግዚአብሔር፡ ልዑል፡ ብሑ</hi>ተ፡ ህሉና፡ ወስልጣን። ዝንቱ፡ ወርኃ፡ ነሐሴ፡
+                                    ወውእቱ፡ ፲ወ፪እምኁልቈ፡ አውራኅ<hi rend="rubric">፨</hi>
```
Comment (thea-m) [inline DabraWarq/DW03.xml:8918]: no, like this in the ms

File: DabraWarq/DW03.xml [hunk 2]
```diff
@@ -9741,7 +9741,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 </msItem>
 
                                 <explicit type="subscriptio" xml:lang="gez">
-                                    <locus from="210rb" to="210va"/> ተፈጸመ፡ ወርኃ፡ ነሐሲ፡ በሰላመ፡ እግዚአብሔር፡
+                                    <locus from="210rb" to="210va"/> ተፈጸመ፡ ወርኃ፡ ነሐሴ፡ በሰላመ፡ እግዚአብሔር፡
```
Comment (thea-m) [inline DabraWarq/DW03.xml:9744]: no, like this in the ms

File: DabraWarq/DW03.xml [hunk 3]
```diff
@@ -9920,12 +9920,12 @@ schematypens="http://relaxng.org/ns/structure/1.0"
 
                                 <explicit type="subscriptio" xml:lang="gez">
                                     <locus target="#211vb"/>
-                                    <hi rend="rubric">ተፈጸመ፡ ወርኃ፡ ጰጕመን፡ ፳ወ፩ አ</hi>ርእስተ፡ ምንባባቲሁ፨ ስብሐት፡
+                                    <hi rend="rubric">ተፈጸመ፡ ወርኃ፡ ጳጕሜን፡ ፳ወ፩ አ</hi>ርእስተ፡ ምንባባቲሁ፨ ስብሐት፡
```
Comment (thea-m) [inline DabraWarq/DW03.xml:9923]: It's actually ጳጕምን in the ms, thanks for spotting the irregularity!

File: DabraWarq/DW03.xml [hunk 4]
```diff
@@ -9920,12 +9920,12 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                                         ለ<hi rend="rubric">ሥሉስ፡ ቅዱስ፡ ለዘጸሐፎ፡ ወለዘአጽ</hi>ሐፎ፡ አቡነ፡ ገብረ፡
                                     ማርያም፡ ጽሢቆ፡ በአስተሐምሞ፡ ከመ፡ ያስተጋብአ፡ ነጊረ፡ ገድሎሙ፡ ለቅዱሳን፡ መስተጋድላን፡ ሰማዕት፡ ወጻድቃን፡ ሠናያነ፡ ግዕዝ፡ መስተዓግሳን፨
-                                አባግዕ፡ ቡሩካን፡ እለ፡ አልቦሙ፡ ጸኒን፡ ኣማን፡ በኣማን፡ ከዋክብት፡ ብሩሃን፡ መኃትዊሃ፡ ለቤተ፡ ክርስቲያን፨ አዕናቈ፡ ክብር፡ ጽዱላን፨ በሃይማኖት፡ ፍጹማን፨ ወከመ፡ ጳውሎስ፡
+                                አባግዕ፡ ቡሩካን፡ እለ፡ አልቦሙ፡ ጸኒን፡ አማን፡ በአማን፡ ከዋክብት፡ ብሩሃን፡ መኃትዊሃ፡ ለቤተ፡ ክርስቲያን፨ አዕናቁ፡ ክብር፡ ጽዱላን፨ በሃይማኖት፡ ፍጹማን፨ ወከመ፡ ጳውሎስ፡
```
Comment (thea-m) [inline DabraWarq/DW03.xml:9926]: no, like this in the ms

File: DabraWarq/DW03.xml [hunk 5]
```diff
@@ -10141,16 +10141,16 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <hi rend="rubric">ስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅ</hi>ዱስ፡ ፩ አምላክ። ን<gap reason="illegible" unit="chars" extent="7"/> ዘ<gap reason="illegible" unit="chars" extent="2"/>
                         <gap reason="illegible" unit="lines" extent="1"/> እምወንጌል፡ ወእምሐዋርያት። እምገድለ፡
                         ሰማዕት፡ ወመነኮስ<gap reason="illegible" unit="chars" extent="2"/>፡ እምእምርሕት፡
-                        ወድርሰናት፨ በረድኤተ፡ እ<hi rend="rubric">ግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ ብሑተ፡</hi> መንግሥት፨
-                        ወበስልጣነ፡ እ<hi rend="rubric">ግዚአብሕር፡ አቡሁ፡ ፈታኔ፡ ሕቡአት። ወ</hi>በሰልጣነ፡ እግዚኣብሔር፡
-                        ጰረቅሊጦስ፡ መጥበቤ፡ ሕፃናት፨ ወበትንብልናሃ፡ ለእግዝእትነ፡ <hi rend="rubric">ማርያም፡</hi> መድኃኒተ፡
+                        ወድርሳናት፨ በረድኤተ፡ እ<hi rend="rubric">ግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ ብሑተ፡</hi> መንግሥት፨
+                        ወበስልጣነ፡ እ<hi rend="rubric">ግዚአብሕር፡ አቡሁ፡ ፈታኔ፡ ሕቡአት። ወ</hi>በስልጣነ፡ እግዚኣብሔር፡
+                        ጳራቅሊጦስ፡ መጥበቤ፡ ሕፃናት፨ ወበትንብልናሃ፡ ለእግዝእትነ፡ <hi rend="rubric">ማርያም፡</hi> መድኃኒተ፡
```
Comment (thea-m) [inline DabraWarq/DW03.xml:10146]: no, line this in the ms

File: DabraWarq/DW03.xml [hunk 6]
```diff
@@ -10141,16 +10141,16 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                         ኵሉ፡ ፍጥረት፨ ወበጸውዖተ፡ ስሞሙ፡ ለ<hi rend="rubric">ሚካኤል፡ ወገብርኤል፡</hi> ልቡሳነ፡ እሳት፨
                         በእንተ፡ ክብሮሙ፡ ለ<hi rend="rubric">፳</hi>ወ<hi rend="rubric">፬</hi> ሊቃናት፨ በእንተ፡
                             <hi rend="rubric">፬</hi> እንስሳ፡ ጸዋርያነ፡ መንበሩ፡ ለጸባኦት፨ ክሥት፡ ሊተ፡ እግዚኦ፡ ከመ፡
-                        እትናገር፡ ምሥጢራት፡ ኵሎሙ፡ መጻሕፍት፤ ዘፈተሕከ፡ አፉሁ፡ ለሙሴ፡ መክብበ፡ ነቢያት፨ ዘኢይትነገር፡ ኅቡአ፡ ጥበብከ፡
-                        አይድዐኒ፡ በከመ፡ ይቤ፡ ዳዊት፨ ወእትናገር፡ ስዉራተ፡ ከመ፡ ኢሳዪያስ፡ ወሕዝቅኤል፡ በላዔ፡ መጻሕፍት፨ ወአስትየኒ፡
+                        እትናገር፡ ምሥጢራት፡ ኵሎሙ፡ መጻሕፍት፤ ዘፈታሕከ፡ አፉሁ፡ ለሙሴ፡ መክብበ፡ ነቢያት፨ ዘኢይትነገር፡ ኅቡአ፡ ጥበብከ፡
+                        አይድዐኒ፡ በከመ፡ ይቤ፡ ዳዊት፨ ወእትናገር፡ ስዉራተ፡ ከመ፡ ኢሳይያስ፡ ወሕዝቅኤል፡ በላዔ፡ መጻሕፍት፨ ወአስትየኒ፡
```
Comment (thea-m) [inline DabraWarq/DW03.xml:10151]: no, like this in the ms

File: DabraWarq/DW03.xml [hunk 7]
```diff
@@ -10141,16 +10141,16 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                         ከመ፡ ዕዝራ፡ ጽዋዐ፡ አእምሮ፡ ዘሕብረ፡ እሳት፨ ወአንብበኒ፡ ሕቡአተ፡ ከመ፡ ዮሐንስ፡ አቡቀለምሲስ፡ ነቢየ፡ ሰብዐ፡
-                        መኃትማት፨ ወምላእ፡ ውስተ፡ ሕሊናየ፡ ዘመንፈስ፡ ቅዱስ፡ እሶት፨ ወጠንኩ፡ አንሰ፡ ለፈጽሞቱ፡ ሀበኒ፡ ረድኤተ፨ ወወሰከኒ፡
+                        መኃትማት፨ ወምላእ፡ ውስተ፡ ሕሊናየ፡ ዘመንፈስ፡ ቅዱስ፡ እሳት፨ ወጠንኩ፡ አንሰ፡ ለፈጽሞቱ፡ ሀበኒ፡ ረድኤተ፨ ወወስከኒ፡
```
Comment (thea-m) [inline DabraWarq/DW03.xml:10153]: እሶት is like this in the ms, ወወሰከኒ was a typo

File: DabraWarq/DW03.xml [hunk 8]
```diff
@@ -10167,7 +10167,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                     <label>12 Maskaram</label>
                     <ab> በዛቲ፡ ዕለት፡ ሠርዑ፡ መምህራነ፡ ቤተ፡ ክርስቲያን፡ ከመ፡ ይግበሩ፡ ለለ፡ ወርኁ፡ አመ፡ ፲ወ፪፡ በዓለ፡ <hi rend="rubric">
                      <persName ref="PRS7049Michael">ሚካኤል፡</persName> ሊቀ፡
-                        መ</hi>ላእክት፡ እስመ፡ በዛቲ፡ ዕለት፡ አሥተርአዮ፡ ለ<persName ref="PRS5584Isaiah">ኢሳዪያስ፡</persName> ነቢይ፨ ወውእቱ፡ ተአዛዚ፡ በዲበ፡ ሠናይቱ፡ ለኵሉ፡ ዘመደ፡ እጓለ፡ እመሕያው፨
+                        መ</hi>ላእክት፡ እስመ፡ በዛቲ፡ ዕለት፡ አሥተርአዮ፡ ለ<persName ref="PRS5584Isaiah">ኢሳይያስ፡</persName> ነቢይ፨ ወውእቱ፡ ተአዛዚ፡ በዲበ፡ ሠናይቱ፡ ለኵሉ፡ ዘመደ፡ እጓለ፡ እመሕያው፨
```
Comment (thea-m) [inline DabraWarq/DW03.xml:10170]: no, like this in the ms

File: DabraWarq/DW03.xml [hunk 9]
```diff
@@ -10182,8 +10182,8 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                     <label>12 Ḫǝdār</label>
                     <ab> በዛቲ፡ ዕለት፡ በዓሉ፡ ለሊቀ፡ መላእክት፡ <persName rend="PRS7049Michael">ሚካኤል፡</persName>
                         መልአከ፡ ኃይል፡ ዘያስተምሕር፡ በእንተ፡ ዘመደ፡ እጓለ፡ እመሕያው፡ </ab>
-                    <ab> ወሄኖኪ።</ab>
-                    <ab>ወእርምዖስኒ፡</ab>
+                    <ab> ወሄኖክ።</ab>
```
Comment (thea-m) [inline DabraWarq/DW03.xml:10185]: actually, ወሄኖክኒ

**Comment by Getinet309** [comment]
Dear Dorothea,
If the ms has an error we can not correct it rather indicating the error. actually to get the full meaning the script should be corrected. I agreed with your comment. but to what reverting is requested? to the first draft or to the other?
Thanks

**Pull request merged**